gradio/routes.py

```diff
@@ -251,6 +251,17 @@ def main(request: fastapi.Request, user: str = Depends(get_current_user)):
                         "the frontend by running /scripts/build_frontend.sh"
                     )
 
+        @app.get("/info/", dependencies=[Depends(login_check)])
```

Comment:
Nit but I think having a query param for `raw: bool` would be more appropriate than two separate routes

Reply:
Also `serialize: bool` may be a better name than raw for the concept of whether we should serialize/deserialize 😂

Reply:
Agreed

Reply:
Fixed

Reply:
Nice!